extension/httpfs/CMakeLists.txt

```diff
@@ -37,6 +36,13 @@ target_link_libraries(kuzu_${EXTENSION_LIB_NAME}_extension
         mbedtls
         ${OPENSSL_LIBRARIES})
 
+#TODO(Tanvir) target_sources is not used in any other makefile
```

Comment:
You moved the crypto utilities from httpfs to the src directory, but none of the functions in src actually use them. As a result, the compiler optimizes out the CryptoUtils class entirely.
Let's keep the crypto module inside the httpfs extension lib.